avro-converter/src/test/java/io/confluent/connect/avro/AvroConverterTest.java

```diff
@@ -156,25 +160,48 @@ public void testNull() {
   }
 
   @Test
-  public void testVersionExtracted() throws Exception {
+  public void testVersionExtractedForDefaultSubjectNameStrategy() throws Exception {
     // Version info should be extracted even if the data was not created with Copycat. Manually
     // register a few compatible schemas and validate that data serialized with our normal
     // serializer can be read and gets version info inserted
+    String subject = TOPIC + "-value";
     KafkaAvroSerializer serializer = new KafkaAvroSerializer(schemaRegistry);
+    AvroConverter avroConverter = new AvroConverter(schemaRegistry);
+    avroConverter.configure(Collections.singletonMap("schema.registry.url", "http://fake-url"), false);
+    testVersionExtracted(subject, serializer, avroConverter);
 
+  }
+
+  @Test
+  public void testVersionExtractedForRecordSubjectNameStrategy() throws Exception {
+    // Version info should be extracted even if the data was not created with Copycat. Manually
+    // register a few compatible schemas and validate that data serialized with our normal
+    // serializer can be read and gets version info inserted
```

Comment:
is this comment relevant here?

Reply:
yes just left it out there since its a test similar to the original one :)